src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,185 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private StorageDao storageDao;
+    private User actualUser;
+    private User actualUser2;

```

Comment:
Numbers in naming is a bad approach

Reply:
Do you really need two users for test? Just asking your opinion about this

Reply:
It's just the only way i see to check if method will behave correctly when there will be more than 1 element. For example if it will throw an exception then such login already exists or return correct user from storage. I've tried to just change actualUser and add it again, but seems like it doesn't work this way.

Reply:
For checking if user with the same login already exists, you could create User, add it to storage.people, create newUser with the same login and call register method, that throws an exception